pymodbus/version.py

```diff
@@ -37,4 +37,4 @@ def __str__(self) -> str:
         return f"[{self.package}, version {self.short()}]"
 
 
-version = Version("pymodbus", 3, 3, 0, "")
+version = Version("pymodbus", 3, 3, 0, "alpha")
```

Comment:
Just for information: can the text be any text e.g. "dev" ?

Reply:
It can't be arbitrary text, but it could be `dev` I think.  Do you want me to try?

https://peps.python.org/pep-0440/#implicit-development-release-number.